src/index.html

```diff
@@ -11,7 +11,169 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="group">
+        <div class="valueDescriptionPair">
+          <label for="surnameField">Surname:</label>
+          <input
+            required
+            type="text"
+            autocomplete="off"
+            id="surnameField"
+            name="surname"
+          >
+        </div>
+        <div class="valueDescriptionPair">
+          <label for="nameField">Name:</label>
+          <input
+            required
+            type="text"
+            autocomplete="off"
+            id="nameField"
+            name="name"
+          >
+        </div>
+        <div class="valueDescriptionPair">
+          <label for="ageField">How old are You?</label>
+          <input
+            min="1"
+            max="100"
+            value="12"
+            type="number"
+            id="ageField"
+            name="age"
+          >
+        </div>
+        <div class="valueDescriptionPair">
+          <label for="dateOfBirdthField">Full date of birth:</label>
+          <input
+            type="date"
+            id="dateOfBirdthField"
+            name="dateOfBirth"
+          >
+        </div>
+        <div class="valueDescriptionPair">
+          <label for="termAgreementCheckBox">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            id="termAgreementCheckBox"
+            name="termAgreement"
+          >
+        </div>
+        <legend>Personal information</legend>
+      </fieldset>
+      <fieldset class="group">
+        <div class="valueDescriptionPair">
+          <label for="eMailField">E-mail:</label>
+          <input
+            required
+            placeholder="email@example.com"
+            type="email"
+            id="eMailField"
+            name="eMail"
+          >
+        </div>
+        <div class="valueDescriptionPair">
+          <label for="passwordField">Password:</label>
+          <input
+            minlength="5"
+            maxlength="10"
+            type="password"
+            id="passwordField"
+            name="password"
+          >
+        </div>
+        <legend>Registration</legend>
+      </fieldset>
+      <fieldset class="group">
+        <div class="valueDescriptionPair">
+          <label for="loveCatFieldYes loveCatFieldNo">Do you love cats?</label>
```

Comment:
You don't need label for that
```suggestion
          Do you love cats?
```